apps/frontend/src/libs/components/slider/slider.tsx

```diff
@@ -0,0 +1,126 @@
+import { Icon } from "~/libs/components/components.js";
+import {
+	useCallback,
+	useEffect,
+	useRef,
+	useState,
+} from "~/libs/hooks/hooks.js";
+
+import { SLIDER_BACKGROUND_COLOR } from "./libs/constants/constants.js";
+import { formatToKebabCase } from "./libs/helpers/helpers.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	label: string;
+	value: number;
+};
+
+const Slider: React.FC<Properties> = ({ label, value }: Properties) => {
+	const MAX = 10;
+	const MIN = 0;
+	const MIN_SCORE_VALUE = 1;
+
+	const initialStep = 0;
```

Comment:
```suggestion
	const INITIAL_STEP = 0;
```
Also it should be not 0, but the real score that user has